code-of-conduct.md

```diff
@@ -1,113 +1,83 @@
+When responding to reports, we may take into account the behavior of members outside formal US-RSE spaces if in our judgment they are relevant to the US-RSE community.
+US-RSE reserves the right to deny or revoke membership to individuals based on past behavior, including conduct towards people who are not members of US-RSE.
+To preserve the time and resources of the Committee, incidents occurring outside US-RSE spaces should only be reported to the Committee if they have a serious impact on the safety of the US-RSE community.
+
+In order to preserve the integrity and health of the Code of Conduct Committee, the Code of Conduct Committee and US-RSE reserve the right to reject any report believed to have been made in bad faith or that is otherwise judged not relevant.
+Intentional false reports or reports that serve as a form of harassment/abuse or to silence legitimate criticism are themselves Code of Conduct violations.
+
+Members of the Code of Conduct Committee who have a genuine conflict of interest or otherwise cannot provide a fair assessment of your report will recuse themselves from handling your report.
+
+All reports to and deliberations of the Code of Conduct Committee will be kept confidential, and we will not publicly name those who report or experience suspected Code of Conduct violations, with the following limited exceptions.
+The US-RSE Code of Conduct and leadership reserves the right to publicize or inform third parties about an incident, including openly naming the offending party, if we believe that doing so is necessary to preserve the reputation and integrity of US-RSE, will increase the safety of US-RSE members, is required by a valid court order or other statutory requirement, or will increase the safety of the general public.
+
+Once a decision has been reached all copies of the complaint and any related notes or material from investigation and deliberation will be destroyed or deleted, with the following exceptions: a summary for internal use to enforce the code of conduct going forward, anonymized as much as is practical, the summary included in USE-RSE’s annual report, and any document that US-RSE is required to retain by a valid court orders or preservation request.
+
+### Potential Response to Code of Conduct Violations
+US-RSE and its Code of Conduct Committee reserve the right to take any action we believe is necessary to protect US-RSE membership or the organization from harm, including reputational and financial harms. Examples of possible responses to code of conduct violations include but are not limited to:
+- Formal warnings
+- Removal of US-RSE responsibilities, privileges, or access to community spaces
+- Permanent of temporary bans from community spaces
+- Permanent or temporary revocation of membership in US-RSE
+Anonymized summaries of  US-RSE Code of Conduct decisions will be included in our semi-annual transparency report whenever possible.
```

Comment:
I don't think a "semi-annual transparency report" has been referenced before this point